[cbase]  can't install

Cbase  is needed for  >misctools.

I have 'this message', Compiling it(I'm on Arm):

libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -Wall -D_REENTRANT -D_POSIX_PTHREAD_SEMANTICS -I. -g -O2 -MT libcbase_la-strings.lo -MD -MP -MF .deps/libcbase_la-strings.Tpo -c strings.c  -fPIC -DPIC -o .libs/libcbase_la-strings.o
strings.c: In function ‘C_string_valist2vec’:
strings.c:376:6: error: wrong type argument to unary exclamation mark
   if(!vp)
      ^
make[2]: *** [Makefile:676: libcbase_la-strings.lo] Error 1
make[2]: *** Waiting for unfinished jobs....
mv -f .deps/libcbase_la-sockctl.Tpo .deps/libcbase_la-sockctl.Plo
make[2]: Leaving directory '/usr/local/tmp/crew/cbase-1.3.7.tar.gz.dir/cbase-1.3.7/lib'
make[1]: *** [Makefile:297: all-recursive] Error 1
make[1]: Leaving directory '/usr/local/tmp/crew/cbase-1.3.7.tar.gz.dir/cbase-1.3.7'
make: *** [Makefile:226: all] Error 2
cbase failed to install: `make V=0 -j2` exited with 2

- Am I, the 'Only one' ?

PS: 
I tested the "cbase-1.3.5.tar.gz" version( found on https://osdn.net/projects/sfnet_slackbuildsdirectlinks/downloads/cbase/cbase-1.3.5.tar.gz/ ), It Is the same message

--- a/packages/cbase.rb
+++ b/packages/cbase.rb
@@ -16,6 +16,8 @@ class Cbase < Package
      x86_64: '2f504105cc4c0aef846e0d5fc272a7baf919517f40cc19a86e55b59c12ab0d61',
   })
 
+  depends_on 'expat'
+  
   def self.build
     # fix error on arm architecture
     case ARCH

--- a/packages/cbase.rb
+++ b/packages/cbase.rb
@@ -3,17 +3,13 @@ require 'package'
 class Cbase < Package
   description 'cbase is a C library of useful functions that simplify systems software development on System V UNIX.'
   homepage 'http://www.hyperrealm.com/oss_cbase.shtml'
-  version '1.3.7-1'
+  version '1.3.7-2'
   source_url 'http://www.hyperrealm.com/packages/cbase-1.3.7.tar.gz'
   source_sha256 'c4d155686ac2e9d1480319de311967fadad745a6ab6971d53d495d9a9e52dc47'
 
   binary_url ({
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/cbase-1.3.7-1-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/cbase-1.3.7-1-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-       i686: '9ce31b80b473652f8da8ede23294d5a8c7fb4b4d9c678d9452a3f8d77a91a8a0',
-     x86_64: '2f504105cc4c0aef846e0d5fc272a7baf919517f40cc19a86e55b59c12ab0d61',
   })
 
   depends_on 'expat'

--- a/packages/cbase.rb
+++ b/packages/cbase.rb
@@ -17,6 +17,14 @@ class Cbase < Package
   })
 
   def self.build
+    # fix error on arm architecture
+    case ARCH
+      when 'armv7l','aarch64'
+        Dir.chdir("lib") do
+          system "sed -i '376c if (&vp == NULL)' strings.c"  # change from   if(!vp)   to    if (&vp == NULL),  tested on armv7l   
+        end
+    end
+    
     system "./configure --prefix=#{CREW_PREFIX}"
     system "make"
   end

